src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(callback, value) {
+      return callback(value);
+    },
+
+    add(value) {
+      calculator.result += value;
+
+      return calculator;
```

Comment:
Instead of returning 'calculator', you should return 'this' to keep the method chainable on the current calculator object.